aspnetcore/signalr/diagnostics.md

```diff
@@ -72,7 +72,7 @@ When using the JavaScript client, you can configure logging options using the `c
 
 [!code-javascript[](diagnostics/logging-config-js.js?highlight=3)]
 
-To disable logging entirely, specify `signalR.LogLevel.None` in the `configureLogging` method.
+To disable logging entirely, specify `signalR.LogLevel.None` in the `configureLogging` method. Note that some logging related to WebSockets are emitted directly by the browser and can't be disabled via setting the log level or any other means.
```

Comment:
I don't think this is specific to WebSockets.

Reply:
I'll fix it.